spa/src/services/auth-manager.ts

```diff
@@ -0,0 +1,15 @@
+import { createRestApi } from "../api/rest-api";
+
+const restApi = createRestApi();
+
+export {
+};
+
+async function startOAuth(onDone: () => void) {
```

Comment:
This is the bit I want in a `manager` layer. It does two things, and I don't want it to pollute our beautiful react code.